server.js

```diff
@@ -7814,6 +7814,58 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Bugzilla bug integration
+camp.route(/^\/bugzilla\/(\d+)\.(svg|png|gif|jpg|json)$/,
+cache(function (data, match, sendBadge, request) {
+  var bugNumber = match[1];  // eg, 1436739
+  var format = match[2];
+  var options = {
+    method: 'GET',
+    json: true,
+    uri: 'https://bugzilla.mozilla.org/rest/bug/' + bugNumber
+  };
+  var badgeData = getBadgeData('bugstatus', data);
+  request(options, function (err, res, json) {
+    badgeData.text[0] = 'Bug ' + bugNumber;
+
+    if (err !== null) {
```

Comment:
Could you change this to use [`checkErrorResponse()`](https://github.com/badges/shields/blob/master/server.js#L552-L555) here, as it helps to lessen code duplication while still covering most cases.

Reply:
Sure thing.